apis/placement/v1beta1/commons.go

```diff
@@ -52,4 +52,8 @@ const (
 	// CRPGenerationAnnotation is the annotation that indicates the generation of the CRP from
 	// which an object is derived or last updated.
 	CRPGenerationAnnotation = fleetPrefix + "CRPGeneration"
+
+	// EnvelopeConfigMapAnnotation is the annotation that indicates the configmap is an envelope configmap that contains resources
+	// we need to apply to the member cluster instead of the configMap itself.
+	EnvelopeConfigMapAnnotation = fleetPrefix + "EnvelopeConfigMap"
```

Comment:
Nit: What is the naming convention? EnvelopeConfigMap or envelope-config-map or envelopeConfigMap? Please state in this doc which format should be used when.

Reply:
Can be addressed in a future PR.

Reply:
is this the naming convention of the annotation or the variable?

Reply:
I was asking about our naming convention for all the label keys and annotation keys in this file. We are using three different formats currently.

Reply:
I see. Will address in another PR.